homework-1/cash_register.py

```diff
@@ -0,0 +1,70 @@
+class Item:
+
+    def __init__(self, name, price):
+        self.name = name
+        self.price = price
+
+
+class CashRegister:
+
+    def __init__(self):
+
+        self.total_price = 0
+        self.discount = 0
+        self.total_items = []
+
+    def add_item(self, item):
+        self.total_items.append(item)
+        print(f"{item.name} has been added to your basket.")
+
+    def remove_item(self, item):
+        self.total_items.remove(item)
+        print(f"{item.name} has been removed from your basket.")
```

Comment:
Might have been nice to consider the case where someone tries to remove an item that wasn't present